packages/malloy-interfaces/thrift/malloy.thrift

```diff
@@ -453,11 +408,13 @@ struct NumberCell {
 struct NullCell {}
 
 struct TimestampCell {
-  1: required string timestamp_value, // TODO another way to represent dates?
+  // TODO another way to represent dates?
+  1: required string timestamp_value,
```

Comment:
Is there a better way to represent dates in Thrift than ISO strings? Epochs?

Reply:
MToy thinks ISO makes sense, because everyone knows what an ISO means (particularly WRT timezones)